package.json

```diff
@@ -59,7 +59,6 @@
     "build": "npm-run-all build:*",
     "build:stats": "ts-node script/stats.ts",
     "build:module": "ts-node script/build-module.ts",
-    "build:nav": "ts-node script/parse-nav.ts",
```

Comment:
What called `build:nav`? Just `npm run build`?

Reply:
Yes just `npm run build`, they use `npm-run-all` to run every script what matches `build:*` regex.